sdk/tests/integration/FileDeleteTransactionIntegrationTests.cc

```diff
@@ -68,7 +68,7 @@ TEST_F(FileDeleteTransactionIntegrationTest, CannotDeleteFileWithNoAdminKey)
   FileId fileId;
   ASSERT_NO_THROW(
     fileId =
-      FileCreateTransaction().setContents({}).execute(getTestClient()).getReceipt(getTestClient()).mFileId.value());
+      FileCreateTransaction().setContents("").execute(getTestClient()).getReceipt(getTestClient()).mFileId.value());
```

Comment:
We can maybe define something like EMPTY_FILE_CONTENT in the File related classes.